tests/v2/validation/token/token_test.go

```diff
@@ -0,0 +1,65 @@
+package token
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	fv3 "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/tokens"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+const (
+	initialTokenDesc = "my-token"
+	updatedTokenDesc = "changed-token"
+)
+
+type TokenTestSuite struct {
+	suite.Suite
+	client  *rancher.Client
+	session *session.Session
+	cluster *management.Cluster
+}
+
+func (t *TokenTestSuite) TearDownSuite() {
+	t.session.Cleanup()
+}
+
+func (t *TokenTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	t.session = testSession
+
+	client, err := rancher.NewClient("", t.session)
+	require.NoError(t.T(), err)
+
+	t.client = client
+}
+
+func (t *TokenTestSuite) TestPatchToken() {
+
+	subSession := t.session.NewSession()
+	defer subSession.Cleanup()
```

Comment:
Since we are using `t.client` to create the token. We don't even need the `subSession`. We only need the sub session if we have multiple test cases, and it needs to cleanup right afterwards, like our integration project tests.

Reply:
Thanks @igomez06! Removed subSession since it's not needed ✅